test/integration/auth_test.go

```diff
@@ -670,12 +670,10 @@ func TestAliceNotForbiddenOrUnauthorized(t *testing.T) {
 // the authentication system but not authorized to do any actions
 // should receive "Forbidden".
 func TestBobIsForbidden(t *testing.T) {
-	deleteAllEtcdKeys()
+	framework.DeleteAllEtcdKeys()
 
 	// This file has alice and bob in it.
-
-	// Set up a master
-	helper, err := master.NewEtcdHelper(newEtcdClient(), testapi.Version(), etcdtest.PathPrefix())
+	helper, err := master.NewEtcdHelper(framework.NewEtcdClient(), testapi.Version(), etcdtest.PathPrefix())
```

Comment:
optional nit: move this line to a NewHelper function in the framework, since it seems to be repeated multiple times.